src/components/Review.jsx

```diff
@@ -1,26 +1,80 @@
 import { useState, useEffect } from "react";
-import {getReviewById} from "../utils/api"
-import {useParams} from "react-router-dom"
-
+import { getReviewById, patchDownVote, patchUpVote } from "../utils/api";
+import { useParams } from "react-router-dom";
+import Comments from "./Comments";
 
 function Review() {
-const [review, setReview] = useState({});
-const [loading, setLoading] = useState([true]);
-const { review_id } = useParams();
+  const [showVote, setShowVote] = useState(true);
+  const [showComments, setShowComments] = useState(false);
+  const [voteStatus, setVoteStatus] = useState("");
+  const [review, setReview] = useState({});
+  const [loading, setLoading] = useState(true);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    getReviewById(review_id).then((reviewData) => {
+      setReview(reviewData);
+      setLoading(false);
+    });
+  }, [review_id]);
 
-useEffect(() => {
-  getReviewById(review_id).then((reviewData) => {
-    setReview(reviewData);
-    setLoading(false);
-  });
-}, [review_id]);
+  function showOnClick() {
+    setShowComments((current) => !current);
+  }
+  function commentsTxt() {
+    if (review.comment_count === 0) {
+      return "There are no comments yet.";
+    } else if (review.comment_count === 1) {
+      return `Show ${review.comment_count} comment`;
```

Comment:
Can the conditional logic be simplified here? Lines 30 and 28 apply for similar conditions